rfcs/2021-11-whcm-support.md

```diff
@@ -0,0 +1,108 @@
+# RFC: Windows High Contrast Mode (WHCM) Support
+
+**Date**: November 29, 2021
+**Status**: `WIP`
+
+## Background
+
+Windows High Contrast Mode (WHCM) is a unique accessibility feature originally designed for Windows users with contrast sensitivity issues. Other reasons a user might enable WHCM include:
```

Comment:
```suggestion
Windows High Contrast Mode (WHCM) is a unique accessibility feature originally designed for Windows users with contrast sensitivity and low vision issues. Other reasons a user might enable WHCM include:
```

There's a non-trivial amount of HCM users with low vision who use it along screen zooming magnification to get content into a perceivable state.